lib/retry.js

```diff
@@ -32,7 +32,8 @@ function afterLaunch(configRetry) {
     */
     var file = resultsFilePath();
     if (!fs.existsSync(file)) {
-        return;
+      /* Result file is not created, because tests passed, consider as success */
```

Comment:
Result file may not be created for other reasons as well. In our case, sometimes it is not created due to Chrome crash - and `protractor-retry` treats that as success. I'm just opened issue #40 on that, just want to point out the possible reason. BTW, it may be the cause for #39 as well.